object-cache.php

```diff
@@ -655,4 +812,73 @@ function __construct() {
 		$this->cache_hits   =& $this->stats['get'];
 		$this->cache_misses =& $this->stats['add'];
 	}
+
+	function increment_stat( $field, $num = 1 ) {
+		if ( ! isset( $this->stats[ $field ] ) ) {
+			$this->stats[ $field ] = $num;
+		} else {
+			$this->stats[ $field ] += $num;
+		}
+	}
+
+	function group_ops_stats( $op, $keys, $group, $size, $time, $comment = '' ) {
+		$this->increment_stat( $op );
+
+		// we have no use of the local ops details for now
+		if ( strpos( $op, '_local' ) ) {
+			return;
+		}
+
+		$this->size_total += $size;
+
+		$strip_keys = apply_filters( 'memcached_strip_keys', true );
+		if ( $strip_keys ) {
+			$keys = $this->strip_memcached_keys( $keys );
+		}
+
+		if ( $time > 0.005 && 'get_multi' != $op ) {
```

Comment:
Should we move this to a class const with a comment around how we arrived at 0.005?

Reply:
It's what we use on WP.com. Will check a bit more.

Reply:
It's `5ms`. Probably based on the [Memcached](https://github.com/memcached/memcached/wiki/Performance#it-should-respond-quickly).